src/Infrastructure/Repository/OperationRepositoryInterface.php

```diff
@@ -2,14 +2,13 @@
 
 declare(strict_types=1);
 
-namespace Bookkeeper\Accounting\Domain\Operation;
+namespace Bookkeeper\Accounting\Infrastructure\Repository;
+
+use Bookkeeper\Accounting\Domain\Operation\Operation;
+use Bookkeeper\Accounting\Domain\Operation\OperationIdInterface;
 
 interface OperationRepositoryInterface
```

Comment:
Repository interfaces used in domain should be in domain. In infrastructe placed implementations.